packages/@pglt/pglt/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "pglt",
+  "version": "0.1.0",
+  "bin": {
+    "pglt": "bin/pglt"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/supabase/postgres_lsp.git",
+    "directory": "packages/@pglt/pglt"
+  },
+  "author": "Philipp Steinrötter",
+  "contributors": [
+    {
+      "name": "Julian Domke",
```

Comment:
Can you change this to supabase community and just list me as a contributor?

Reply:
sure!